compliance_checker/cf/cf.py

```diff
@@ -148,9 +149,11 @@ def _find_cf_standard_name_table(self, ds):
 
             self._std_names = util.StandardNameTable(location)
             return True
-        except Exception:
+        except Exception as e:
             # There was an error downloading the CF table. That's ok, we'll just use the packaged version
-            print("Error fetching standard name table. Using packaged v{0}".format(self._std_names._version))
+            msg = ("Problem fetching standard name table:\n{0}\n"
+                   "Using packaged v{1}").format
+            warn(msg(e, self._std_names._version))
```

Comment:
this is a curious approach - since you don't reuse msg anywhere, why store the ref to `format` and call it as opposed to making msg the template string and and calling `format` in the warn line?